writing/automation.md

```diff
@@ -0,0 +1,27 @@
+# Automate Everything
+
+---
+
+- Time is valuable, don't do things computers can do for you. Remembering (calendar, checklists, reminders). Free yourself for more creative thinking and enjoying life
+- As programmer, goal is to build scripts so you don't have to do anything
+  - Automating yourself out of a job
+- Bringing into life
+
+  - Checklists and reminders through Butler
```

Comment:
I'm an example comment saying. Well done, this note is great. However instead of this line you could write: 
```suggestion
  - Checklists and reminders through Butler is the best thing ever!!!
```